### PR TITLE
Added more Hai news

### PR DESCRIPTION
**Pull request by BlazingDiesel**
**Content (News)**

## Summary
PR for adding some more spaceport news in Hai systems. This includes news in preexisting categories, news in new categories, news from Hai, news after TMBR missions and Greenwater outfitter, and holographic advertisements for both humans and Hai. Overall, just a greater quantity and variety of Hai news. Any additions/changes that one feels could be made are appreciated.

### CODE REVIEW

File: data/hai/hai news.txt [hunk 1]
```diff
@@ -91,6 +99,74 @@ news "cute story in Hai space"
 			"I saw the Hai-human fusion band Nuclear Martini playing on Luna. Nobody realized the hairy drummer was really a Hai in disguise."
 			"I saw a Hai wearing the cutest bonnet the other day. When I get back to human space, I'm definitely getting a pet squirrel."
 			"Do you see that Hai woman carrying her baby? Doesn't it look like a long-tailed teddy-bear? The Hai are so cute."
+			"Do you ever wonder what the Hai would think if they saw an actual squirrel? Do they even know that squirrels exist? Are squirrels just miniature versions of the Hai? I'm so lost."
+			"I always felt so rejected working as a servant on Shangri-la. Ever since I bought a ticket and ended up in Hai space, everyone around me has made me feel like I'm worth something."
```
Comment (MasterOfGrey) [inline data/hai/hai news.txt:103]: Minor point - bought from whom? It's hard to end up here accidentally or easily without your own ship,

File: data/hai/hai news.txt [hunk 2]
```diff
@@ -91,6 +99,74 @@ news "cute story in Hai space"
...
+	name
+		word
+			"Hai civilian"
+			"Hai tour guide"
+	message
+		word
+			`"`
+		word
+			"Humans are always so curious when they discover our space past the wormhole. It is truly our pleasure to welcome them all and be amongst them."
+			"The little bit of hair that humans have on the tops of their head are so cute. I love how many ways they style it."
```
Comment (MasterOfGrey) [inline data/hai/hai news.txt:121]: The hair is the subject.
```suggestion
			"The little bit of hair that humans have on the top of their head is so cute. I love how many ways they style it."
```

File: data/hai/hai news.txt [hunk 3]
```diff
@@ -114,6 +190,24 @@ news "concerned merchant in Hai space"
...
+			"Hai merchant"
+	message
+		word
+			`"`
+		word
+			"Be careful should you choose to fly up north. Too many human captains who don't know any better become victims to those Hai seeking great trouble."
+			"It would be smart to follow trade routes that only visit Hai and human space. One would not wish to experience the business of our northern brethren."
+			"Seeing so many human ships come by with such extensive weapons feels very reminiscent of past times, unfortunately."
+			"Rations are very often sent to those starving to the north of here, but such actions are very often not met with similar kindness. Take such a task very carefully."
+			"I fear what may happen to those who attempt to disrupt the Quarg to the south. They do stay out of the way, but do become retaliatory if initiated."
```
Comment (MasterOfGrey) [inline data/hai/hai news.txt:207]: The Hai probably wouldn't have fired on the Quarg in nearly 180,000 years. I wouldn't say this one is particularly sensible to include.

File: data/hai/hai news.txt [hunk 4]
```diff
@@ -132,6 +226,45 @@ news "human citizen in Hai space"
...
+	name
+		word
+			"Businiessperson"
+			"Professional investor"
+			"Retiree"
+	message
+		word
+			`"`
+		word
+			"Real estate values for Greenwater homes have skyrocketed in recent months, so I'm taking advantage of this opportunity as much as I can!"
```
Comment (MasterOfGrey) [inline data/hai/hai news.txt:262]: This makes the assumption that the Hai allow US style price speculation on property and renter's markets. Something which is a legacy of rapid population growth and immigration. This isn't even true of all of our world, let along an alien world that seems to have resolved the majority of its economic difficulties and has been declining in population slowly for longer than humans have had fire.

**Review by MasterOfGrey — COMMENTED** [review]
The overwhelming majority of these are excellent, just a couple of comments here and there.

File: data/hai/hai news.txt (hunk 1 above)
Comment (BlazingDiesel) [reply data/hai/hai news.txt:103]: Yeah, that makes sense. I suppose that I could modify it to say they bought it from "a shady guy offering to move people far away no questions asked" or maybe just acquired a bunk from "a hippy who wanted to visit a land with giant squirrels". I'd prefer not to remove the entry entirely, but if I can't find a way to make it work I may.

File: data/hai/hai news.txt (hunk 2 above)
Comment (W1zrad) [reply data/hai/hai news.txt:121]: Yup, that needs to change. There's some strangeness with the second possible suggestion: I can't quite put my finger on it right now, but I think something needs to be changed. But I'd just go with the first of these two anyways; it feels more natural to me.

File: data/hai/hai news.txt [hunk 5]
```diff
@@ -114,6 +190,23 @@ news "concerned merchant in Hai space"
...
+		word
+			"Hai merchant"
+	message
+		word
+			`"`
+		word
+			"Be careful should you choose to fly up north. Too many human captains who don't know any better become victims to those Hai seeking great trouble."
+			"It would be smart to follow trade routes that only visit Hai and human space. One would not wish to experience the business of our northern brethren."
+			"Seeing so many human ships come by with such extensive weapons feels very reminiscent of past times, unfortunately."
+			"Rations are very often sent to those starving to the north of here, but such actions are very often not met with similar kindness. Take such a task very carefully."
```
Comment (roadrunner56) [inline data/hai/hai news.txt:206]: Reducing some word repetition. 
```suggestion
			"Rations are very often sent to those starving to the north of here, but such actions are almost never met with similar kindness. Take such a task very carefully."
```

File: data/hai/hai news.txt [hunk 6]
```diff
@@ -132,6 +225,44 @@ news "human citizen in Hai space"
...
+			"Merchant captain"
+			"Engineer"
+			"Merchant pilot"
+	message
+		word
+			`"`
+		word
+			"I've been stranded here for ages waiting for the right parts to fix my ship. Finally I don't have to stay at that dank and musty hotel."
+			"I was weighting the need for Hai technology on my ship due to their high prices. Thankfully I've been able to outfit it with human tech so I don't have to pay such great amounts!"
+			"I had a good thing going before this outfitter showed up. I could transport human ship parts before there was any competition at all, now my services are practically useless!"
```
Comment (roadrunner56) [inline data/hai/hai news.txt:245]:
```suggestion
			"I had a good thing going before this outfitter showed up. I could transport human ship parts before there was any competition at all. Now my services are practically useless!"
```

File: data/hai/hai news.txt [hunk 7]
```diff
@@ -132,6 +225,44 @@ news "human citizen in Hai space"
...
+			"Businiessperson"
+			"Professional investor"
+			"Retiree"
+	message
+		word
+			`"`
+		word
+			"None of the economic models that I've run can explain why the walnut market never stops growing. Sure they're pretty popular, but this has to be a bubble, right? Right?!"
+			"I've noticed a rise in the number of human cuisine-focused establishment in this area. Hopefully a sizable investment in food imports could help my portfolio recover."
+			"I think I can coast for a while off my Darkmetal mine investments, they never seem to fail me. Maybe I can even get that crazy Hai motorcycle I've been eyeing up soon."
```
Comment (roadrunner56) [inline data/hai/hai news.txt:263]:
```suggestion
			"I think I can coast for a while off my Darkmetal mine investments; they never seem to fail me. Maybe I can even get that crazy Hai motorcycle I've been eyeing up soon."
```

File: data/hai/hai news.txt [hunk 8]
```diff
@@ -165,3 +296,36 @@ news "reformed pirate in Hai space"
...
+	location
+		government "Hai"
+		attributes "hai tourism"
+	name
+		"Holographic advertisement"
+	message
+		word
+			`"`
+		word
+			"Are regular tree nuts too very normal and bland? Ibrahim's Exotic Nut Shack fulfills all your many desires for exciting, flavorful nut varieties."
```
Comment (roadrunner56) [inline data/hai/hai news.txt:327]:
```suggestion
			"Are regular tree nuts too normal and bland? Ibrahim's Exotic Nut Shack fulfills all your many desires for exciting, flavorful nut varieties."
```

**Review by roadrunner56 — CHANGES_REQUESTED** [review]
Small grammar and punctuation fixes. I really like this PR and how it fleshes out Hai society a bit more.

**Comment by BlazingDiesel** [comment]
At this point I've changed the bits that others have mentioned. I also (in an accidental attempt to update the more-hai-news branch to the latest commits on the master branch) pushed another commit from the master, apologies if this disrupts anything.

**Review by roadrunner56 — APPROVED** [review]
Full disclaimer, I can't comment on the code side of the PR, but the content side looks great! Congrats on your first PR!

**Comment by MasterOfGrey** [comment]
The `Merge remote-tracking branch 'upstream/master' into <branch name>` commit merely aligns you to the current master state, which helps with some things and makes sure you're on top of any arising merge conflicts.

You will notice when you look at the "Files Changed" tab for this PR, it is still only your changes shown, so there is no issue.

File: data/hai/hai news.txt [hunk 9]
```diff
@@ -187,6 +230,8 @@ news "concerned merchant in Hai space"
 			"The Hai don't like my ship's armory of laser rifles, but I was once boarded by a pirate crew and I'm never letting that happen again! If some overgrown squirrel is going to take away my rifle, it'll have to be from my cold dead hands!"
 			"I hope nobody is smuggling Hai technology out of here. I don't want to think about what would happen if the pirates got their hands on this stuff."
 			"The Hai will pay you to rush cargo to human space and back. Don't die trying."
+			"I really want to go somewhere else but here, but I feel like I'm sandwiched between the Hai to the north and the pirates past the wormhole."
```
Comment (roadrunner56) [inline data/hai/hai news.txt:233]:
```suggestion
			"I really want to go somewhere else other than here, but I feel like I'm sandwiched between the Hai to the north and the pirates past the wormhole."
```

File: data/hai/hai news.txt [hunk 10]
```diff
@@ -204,6 +250,7 @@ news "concerned hai in Hai space"
 			"It would be smart to follow trade routes that only visit Hai and human space. One would not wish to experience the business of our northern brethren."
 			"Seeing so many human ships come by with such extensive weaponry feels very reminiscent of past times, unfortunately."
 			"Rations are very often sent to those starving to the north of here, but such actions are almost never met with similar kindness. Take such a task very carefully."
+			"I feel so terrible for the many humans that attempt to reach our worlds and are destroyed by the Unfettered humans that block their passage."
```
Comment (roadrunner56) [inline data/hai/hai news.txt:253]: Aren't unfettered humans Alphas? I don't think Alphas are hanging around the Hai wormhole.

**Review by roadrunner56 — CHANGES_REQUESTED** [review]

File: data/hai/hai news.txt (hunk 10 above)
Comment (BlazingDiesel) [reply data/hai/hai news.txt:253]: In one of the missions where pirates go through the wormhole, the Hai refer to them as 'Unfettered humans', so I felt it was appropriate to refer to them the same here.

Comment (roadrunner56) [reply data/hai/hai news.txt:253]: Ok then, feel free to resolve.

**Review by roadrunner56 — APPROVED** [review]

File: data/hai/hai news.txt [hunk 11]
```diff
@@ -165,3 +346,41 @@ news "reformed pirate in Hai space"
...
+			"Holographic advertisement"
+	message
+		word
+			`"`
+		word
+			"Are regular tree nuts too normal and bland? Ibrahim's Exotic Nut Shack fulfills all your many desires for exciting, flavorful nut varieties."
+			"Come to relax inside the many tranquil hot springs of Mirrorlake by visiting the great paradise of Freezewarm Resort & Spa."
+			"Professional full-body fur stylists are available for hire at the nearby Eliana's Styling Center! Acquire a new and fashionable look, or even dye your fur coloring!"
+			"The Hai Tourism Department recommends visiting Frostmark very soon, while the warm and rainy summer experience is still taking place!"
+			"If you crave bizarre cuisine from beyond the wormhole, you must try the food at Duality Diner, featuring human chef Dibani Wilczek and her excellent dishes like 'broccoli', 'oranges', and 'cereal'!"
```
Comment (Amazinite) [inline data/hai/hai news.txt:384]:
```suggestion
			"If you crave bizarre cuisine from beyond the wormhole, you must try the food at Duality Diner, featuring human chef Dibani Wilczek and her excellent dishes like 'broccoli,' 'oranges,' and 'cereal'!"
```

**Review by Amazinite — CHANGES_REQUESTED** [review]

**Review by Amazinite — APPROVED** [review]

**Pull request merged**